saltboot/hostname.go

```diff
@@ -134,7 +144,7 @@ func updateHostsFile(hostName string, domain string, file string, ipv4address st
 	return nil
 }
 
-func updateSysConfig(hostName string, domain string, file string) error {
+func updateSysConfig(hostName, domain string, file string) error {
```

Comment:
The second `string` could also be removed:
```suggestion
func updateSysConfig(hostName, domain, file string) error {
```